packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1527,6 +1529,31 @@ def submit_user_code_to_user_code() -> list[Callable]:
     ]
 
 
+@transform(UserCode, SubmitUserCode)
+def user_code_to_submit_user_code() -> list[Callable]:
+    return [
+        rename("raw_code", "code"),
+        rename("service_func_name", "func_name"),
+        keep(
+            [
+                "id",
+                "code",
+                "func_name",
+                "signature",
+                "input_policy_type",
+                "input_policy_init_kwargs",
+                "output_policy_type",
+                "output_policy_init_kwargs",
+                "deployment_policy_type",
+                "deployment_policy_init_kwargs",
+                "input_kwargs",
+                "enclave_metadata",
```

Comment:
This was part of the old Enclave implementation and is now redundant. We can remove it.

Reply:
ah ok, this is specifically `enclave_metadata` that can be removed?

Reply:
Yep 💯